src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static int getRandomNumber() {
+        return new Random().nextInt(101);
+    }
+
+    public static Ball getRandomBall() {
```

Comment:
Please read common mistakes more cerafully (https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-use-static-methods-in-your-solution)